docs/how-to-guides/access-monitoring.md

```diff
@@ -0,0 +1,82 @@
+# Access Monitoring on Lokomotive
+
+## Contents
+
+* [Introduction](#introduction)
+* [Presumptions](#presumptions)
+* [Prometheus](#prometheus)
+* [Alertmanager](#alertmanager)
+* [Grafana](#grafana)
+  * [Using Port Forward](#using-port-forward)
+  * [Using Ingress](#using-ingress)
+
+## Introduction
+
+You can deploy the monitoring stack shipped with lokomotive as shown in the Prometheus Operator component [guide](../configuration-reference/components/prometheus-operator.md).
+
+## Presumptions
+
+- Prometheus Operator component is deployed in `monitoring` namespace. If you have specified a different namespace in config then change the commands accordingly, `kubectl -n <your namespace> ...`.
+
+## Prometheus
+
+Run following command to port forward Prometheus locally on port `9090`.
+
+```
+kubectl -n monitoring port-forward svc/prometheus-operator-prometheus 9090:9090
+```
+
+Now goto [http://localhost:9090](http://localhost:9090).
+
+## Alertmanager
+
+Run following command to port forward Alertmanager locally on port `9093`.
+
+```
+kubectl -n monitoring port-forward svc/prometheus-operator-alertmanager 9093:9093
+```
+
+Now goto [http://localhost:9093](http://localhost:9093).
```

Comment:
Is `Now goto` the correct words here ? Perhaps something like `Access the URL` ?

Reply:
Maybe `Now open the following URL in your browser?`